arm_robot:
  arm_robot_trajectory_controller:
    type: position_controllers/JointTrajectoryController
    joints:
      - arm1_joint
      - arm2_joint
      - arm3_joint
      - arm4_joint
      - arm5r_joint
    gains:
      arm1_joint: {p: 100, d: 1, i: 1, i_clamp: 1}
      arm2_joint: {p: 100, d: 1, i: 1, i_clamp: 1}
      arm3_joint: {p: 100, d: 1, i: 1, i_clamp: 1}
      arm4_joint: {p: 100, d: 1, i: 1, i_clamp: 1}
      arm5r_joint: {p: 100, d: 1, i: 1, i_clamp: 1}

  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50
